buildscripts/cspc-operator-debug/Dockerfile

```diff
@@ -0,0 +1,30 @@
+#
+# This Dockerfile builds a recent cspc-operator using the latest binary from
+# cspc-operator  releases.
+#
+FROM alpine:3.6
+
+RUN apk add --no-cache \
+    iproute2 \
+    bash \
+    curl \
+    net-tools \
+    mii-tool \
+    procps \
+    libc6-compat \
+    ca-certificates
+
+COPY cspc-operator /usr/local/bin/cspc-operator
```

Comment:
nit : Can we different binary name ? or look at the above comment to have different directories

Reply:
done